utils/unittest/logging.go

```diff
@@ -39,6 +42,16 @@ func LoggerWithWriterAndLevel(writer io.Writer, level zerolog.Level) zerolog.Log
 	return log
 }
 
+// go:noinline
+func LoggerForTest(t *testing.T, level zerolog.Level) zerolog.Logger {
+	_, file, _, ok := runtime.Caller(1)
```

Comment:
Should `t.Helper()` be called in here?

Reply:
TIL: `t.Helper()` =)

I think we are fine here, since we are not (and would not be) calling any flow-control test functions (e.g. t.Skip/t.Fail/etc) from this function, so user won't ever see this function in the testing output.